packages/eslint-plugin/docs/rules/no-empty-function.md

```diff
@@ -44,4 +44,36 @@ See the [ESLint documentation](https://eslint.org/docs/rules/no-empty-function)
 }
 ```
 
+## Options
+
+This rule has an object option:
+
+- `allow` (`string[]`)
+  - `"protected-constructors"` - Protected class constructors.
+  - `"private-constructors"` - Private class constructors.
```

Comment:
nit, would probably be good to point the user toward the "extra" options
```suggestion
  - `"private-constructors"` - Private class constructors.
  - [See the other options allowed](https://github.com/eslint/eslint/blob/master/docs/rules/no-empty-function.md#options)
```

Reply:
Yes, you are right. Thanks.